stacks/SiriGeneratorStack.ts

```diff
@@ -127,6 +140,16 @@ export const SiriGeneratorStack = ({ stack }: StackContext) => {
 
     validatorBucket.addEventNotification(EventType.OBJECT_CREATED, new LambdaDestination(siriValidator));
 
+    siriValidator
```

Comment:
Can we check if this metric is needed because in the `MonitoringStack.ts` file there already is a `SiriValidationAlarm` that I think is performing the same function

Reply:
removed